devel/lazarus/Portfile

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                lazarus
+version             2.0.4
+revision            1
```

Comment:
Since this is the first version of the package, it would be `revision 0`

Reply:
yep. my fault.